jupyterhub_moss/spawner.py

```diff
@@ -228,33 +234,38 @@ async def _get_partitions_info(self) -> dict[str, PartitionInfo]:
         2. Parses output with slurm_info_resources
         3. Combines info with partitions traitlet
         """
-        # Execute given slurm info command
-        subvars = self.get_req_subvars()
-        cmd = " ".join(
-            (
-                format_template(self.exec_prefix, **subvars),
-                format_template(self.slurm_info_cmd, **subvars),
+        partitions = PartitionsTrait.model_validate(self.partitions)
+        partitions_info = {}
+
+        # Execute given slurm info command on each partition
```

Comment:
how often does this get run? Because if I understand it, instead of running on `sinfo`, you now run one for every configured partition?

Reply:
indeed, there is one `sinfo` per partition now. However, it only loops once for every landing on the resource selection page. Clicking around or switching between _simple_/_advanced_ tabs does not trigger any more `sinfo` commands.